sotodlib/hwp/hwp.py

```diff
@@ -160,6 +161,8 @@ def get_hwpss(aman, signal=None, hwp_angle=None, bin_signal=True, bins=360,
         hwpss_stats.wrap('redchi2s', redchi2s, [(0, 'dets')])
 
     else:
+        if isinstance(flags, str):
+            flags = aman.flags.get(flags)
```

Comment:
I think a more flexible option for the string input option is to use `attrgetter` here's [an example](https://github.com/simonsobs/sotodlib/blob/88cbf7a9d955485553b4da3b06e13fbcce2fbad5/sotodlib/tod_ops/filters.py#L380-L381). This allows you to search for flags in a field buried in the axis manager i.e. `aman.turnaround.turnaround_flags` you could pass as a string `turnaround.turnaround_flags` and `attrgetter` will work through the layers and then doesn't require the `RangesMatrix` to be in `aman.flags`

Reply:
attrgettr is not necessary because aman supports dot separated path https://github.com/simonsobs/sotodlib/pull/1057
I will change to `aman.get(flags)` instead.

Reply:
Oh ya I had forgotten that that was added, great!